public/app/themes/justice/inc/admin.php

```diff
@@ -36,4 +37,41 @@ public static function removeCustomizer()
         $customizer_url = add_query_arg('return', urlencode(remove_query_arg(wp_removable_query_args(), wp_unslash($_SERVER['REQUEST_URI']))), 'customize.php');
         remove_submenu_page('themes.php', $customizer_url);
     }
+
+    public static function removePublishPressUpsellBranding()
+    {
+        echo '<style>
+               .toplevel_page_revisionary-q {
+                 li:last-of-type {
+                   display: none;
+                 }
+               }
+               .pp-version-notice-bold-purple {
+                 display: none;
+               }
+               .pressshack-admin-wrapper {
+                 a {
+                  color: #2271b1;
+                  &:hover {
+                    color: #135e96;
+                  }
+                  &:focus {
+                    color: #043959;
+                  }
+                }
+                 .inner-sidebar {
+                   display: none;
+                 }
+                 footer {
+                   display: none;
+                 }  
+               }
+               .ppseries-settings-body-content {
+                 width: 100% !important;
+               }
+               .rvy-creation-ui {
```

Comment:
I think removing the UI for Submit or Approve has the unintended consequence of the revisions sitting in the Revision Queue and being in the Not Submitted state.

Reply:
I'm not sure it makes a difference to the flow? I'm not removing submit/approve here, only Preview/Edit.

The flow in both cases is:
1. Make a new revision
2. Preview revision
3. Submit and approve from the preview/edit page

Reply:
When I check locally, it's removing the Submit / Approve UI.

<img width="297" alt="image" src="https://github.com/ministryofjustice/justice-gov-uk/assets/15802017/45a15de3-f4c1-4886-958e-296fd987bc93">

To reproduce, browse the frontend and click 'New Revision' in the top bar. Or in admin, All Pages -> hover a page -> New Revision.

I wonder if we could fix the bug, maybe there's a conflict with the theme or another plugin.

Reply:
Fixed with the latest version of the plugin